queries/css/folds.scm

```diff
@@ -0,0 +1,3 @@
+[
+  (rule_set)
```

Comment:
I'm a bit confused if we should use `block` or `rule_set`. I personally prefer it to be folded based on `rule_set` but I don't know if anyone other than me prefer it this way

Reply:
As you are the first one adding this, we'll go for your version, and enhance afterwards.